content/en/blog/falcosidekick-response-engine-part-8-fluxv2.md

```diff
@@ -0,0 +1,346 @@
+flux create source git alpine \
+    --url https://github.com/developer-guy/desired-state-repository \
+    --branch master \
+    --interval 30s
+```
+
+To create `Kustomization`:
+
+```shell
+flux create kustomization alpine \
+    --source alpine \
+    --path "./" \
+    --prune true \
+    --validation client \
+    --interval 5m
+```
+
+### Install falco-event-listener
+
+As the name suggests, this program will listen to an event in the form of `CloudEvents.` This CloudEvents forwarded from Falcosidekick, a simple daemon for enhancing available outputs for Falco. After the successful receipt of the event, `falco-event-listener` will update the YAML definition to scale its replicas to zero based on the pieces of information given in the event.
+
+In most basic form, the architecture of the demo is:
+
+Falco w/HTTP --> Falcosidekick w/CloudEvent --> falco-event-listener w/HTTP --> GitHub
+
+To learn more about `CloudEvents`, see [cloudevents.io](https://cloudevents.io).
+
+First, let us clone the falco-event-listener repository.
+
+```shell
+git clone https://github.com/developer-guy/falco-event-listener
+cd falco-event-listener
+```
+
+Before installing this project, we have to do a couple of things.
+
+* We have to [create a GitHub PAT (Personal Access Token)](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token) to be able to update the desired state configurations which are in the GitHub repository after we detect a malicious behavior related to our pod has detected.
+
+As you can see in the above arguments, we should pass `GITHUB_TOKEN` as an argument to our CLI application. The best option is storing `GITHUB_TOKEN` in a Kubernetes Secret and [using Secret as environment variables](https://kubernetes.io/docs/concepts/configuration/secret/#using-secrets-as-environment-variables).
+
+```shell
+kubectl create namespace falco
+kubectl create secret generic github-secret \
+         --from-literal GITHUB_TOKEN=$GITHUB_TOKEN \
+         --namespace falco
+````
+
+* You might notice that we are using some URLs (--notify-url) within the flags of the project. Let me explain that why we need it a bit. As you might know, we are trying to set up some remediation engine here, so we have to react to those events thrown by the Falcosidekick as soon as possible. We defined an interval while creating a `GitRepository`, which means that Flux v2 will wait at least that long to sync configurations, so we have to notify Flux controllers about changes once we edited the desired state by a process whose name is `falco-event-listener.` To notify the Flux controllers about changes in Git or Helm repositories, we can set up webhooks and trigger a cluster reconciliation every time a source changes. For more detail, please [see](https://fluxcd.io/docs/guides/webhook-receivers/).
+
+There are different kind of webhook receivers in Flux v2 but we'll use the `generic` one in this guide.
```

Comment:
```suggestion
There are different kinds of webhook receivers in Flux v2, but we'll use the `generic` one in this guide.
```

Reply:
thank you so much for fixing the issues, it should be fixed now 🙏